docs/logging.rst

```diff
@@ -41,6 +41,17 @@ The specified level can be either a string or an integer:
 | NOTSET   | 0             |
 +----------+---------------+
 
+Adjusting Log-Levels from python
+================================
+
+If you prefer, you can also adjust the logging level from python when
+running an experiment by passing the long version of the log level
+command line parameter as an option;
```

Comment:
`;` is a typo, right?

Reply:
I think it would be good to mention that this only works when running the experiment with `Experiment.run`, and not when using `Experiment.main` or `Experiment.automain`.

Reply:
> `;` is a typo right?

No, it was intentional, just perhaps not the best English grammar :D

> I think it would be good to mention that this only works when running the experiment with `Experiment.run`, and not when using `Experiment.main` or `Experiment.automain`.

A very good point - I wasn't aware of this. I'll update the PR.